django_lightweight_queue/cron_scheduler.py

```diff
@@ -134,11 +137,15 @@ def get_matcher(minval, maxval, t):
             row['sigkill_on_stop'] = row.get('sigkill_on_stop', False)
             config.append(row)
 
-            # We must ensure we have at least one worker for this queue.
-            app_settings.WORKERS.setdefault(row['queue'], 1)
-
     return config
 
+
+def ensure_queue_workers_for_config(config):
+    for row in config:
+        # We must ensure we have at least one worker for this queue.
```

Comment:
a5743f9; I realised that the comment is actually also wrong, hopefully the new docstring is clearer.